### ISSUE TITLE
Symlink for the .vimrc file gets rejected

### ISSUE THREAD

**Issue by physicophilic** [terrychou/iVim#104]
I am using iPadOS 13.2.1, and it is jailbroken. I am working from /var/mobile/Documents directory,
as I have unsandboxed iVim, succesfully. 

In order to sync the .vimrc file with the rest of my devices I was trying to create a symlink, but after
having tried both soft and hard links, I gave up.

Each time when I would open Vim it would show black screen and wouldn't launch. This kind of symlinking works perfectly on my other devices. As soon as I put back the original .vimrc file in this folder, it starts up. 

What is going wrong? Can you help?

**Comment by terrychou** [comment]
have you updated your $HOME to reflect the folder of .vimrc?

**Comment by physicophilic** [comment]
Originally $HOME is ~/ directory which is /var/mobile by default. After unsandboxing, Vim automatically took
/var/mobile/Documents as its "home" directory, i.e. it  responded to a vimrc present in this folder. I tried placing .vim and .vimrc outside to ~/, but it doesn't respond to that.

After reading your comment, I tried to change my $HOME to /var/mobile/Documents by editing /etc/passwd as root, but not only it didn't work, my other unsandboxed apps stopped working, so I changed it back. I followed this [link](https://websistent.com/change-home-directory-in-linux/) to do this, but in iOS instead of having 

`username:x:500:500::/home/new_home_directory:/bin/bash` 
like code which is found in linux, there is stuff like `username:/smx7MTQIi2M:...`  (see the image of /etc/passwd)

![unnamed](https://user-images.githubusercontent.com/57056860/78661749-ece0ec80-78ec-11ea-98c1-144c470f2516.jpg)
 
What to do?

**Comment by terrychou** [comment]
You don't have to change your $HOME, you can just put the `.vimrc` file into it.

Are you sure we are talking about `iVim` of this project?

**Comment by physicophilic** [comment]
I am sorry I wasn't able to reply back then. I stopped worrying about it, but now that I am on github I want to reply back to you. I came up with alternative methods, but it would be mcuh better if iVim can handle this. 

At first I did understand you wrong. However, yes, all mentions were to iVim of this project.

I wrongly thought $HOME is a bash variable  (~/), and hence (as per your first reply) tried to change it to current home of vim ~/Documents. The above posted image is me trying to use terminal vim to do that - which is probably why you asked about I'm talking of iVim.  

Later I understood that it's Vim's variable, and was ~/Documents all along. After un-sandboxing, Vim automatically took this to be the $HOME.  So answer to your first post is  - yes, the .vimrc was always in $HOME. The blackscreen occurs as described.

Thanks for vigilantly checking on the issues.
